test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,57 @@
+package seedu.addressbook.commands;
+
+import org.junit.Before;
+import org.junit.Test;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.tag.UniqueTagList;
+
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+public class SortCommandTest {
+    private Person aliceBetsy;
+    private Person bobChaplin;
+
+    private Tag tagMathematician;
+
+    private AddressBook addressBook;
+
+    @Before
+    public void setUp() throws Exception {
+        tagMathematician = new Tag("mathematician");
+        aliceBetsy     = new Person(new Name("Alice Betsy"),
+                new Phone("91235468", false),
```

Comment:
You can avoid using magic numbers here. If you look through the test files, you will be able to find some dummy data of `Person` which you can use to test or you can create MAGIC_VARIABLES in this file and use it rather than directly putting the value here